src/components/Hero/Hero.styles.ts

```diff
@@ -30,15 +30,15 @@ export const SectionStyled = styled.section<TStyleProps>`
     overflow: hidden;
     position: relative;
     width: 100%;
-    
```

Comment:
did you accidentally add white space in this file a few times @Maki44 could you remove it please? Its not bad but its not part of this task so its better to do clean / refactor up tasks separately

Reply:
I must be, that file does not have anything to do with the upload. My bad will remove the white space.